example/config.ts

```diff
@@ -11,27 +20,38 @@ interface GeoJsonConfig<P = any> extends Example {
   activeStyle?: PathOptions
 }
 
+interface WfsConfig<P = any> extends Example {
+  options?: GeoJSONOptions<P>
+  zoomLevel: ZoomLevel
+}
+
 interface NonTiledConfig extends Example {
   options: WMSOptions
 }
 
 const MAP_API_ROOT = 'https://map.data.amsterdam.nl/'
 
+const markerStyle = {
```

Comment:
Perhaps rename to geoJSONStyle or sth? Now it seems it's style for a marker.

Reply:
it is a marker style, geoJSON is encryption format